api/http/handler/handler.go

```diff
@@ -91,6 +93,8 @@ func (h *Handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		http.StripPrefix("/api", h.TeamMembershipHandler).ServeHTTP(w, r)
 	case strings.HasPrefix(r.URL.Path, "/api/websocket"):
 		http.StripPrefix("/api", h.WebSocketHandler).ServeHTTP(w, r)
+	case strings.HasPrefix(r.URL.Path, "/api/webhook"):
```

Comment:
Whilst we use `webhook` in the bolt package, we'll use `webhooks` in the http package. Could you change it to `/api/webhooks` ?